src/components/translate/TranslationSentenceStatus.vue

```diff
@@ -91,12 +95,12 @@ export default {
     version() {
       this.getStatusData();
     },
+    translateStatus() {
+      this.getStatusData();
+    },
   },
   mounted() {
-    this.update = setTimeout(() => { this.getStatusData(); }, 5);
-  },
-  beforeDestroy() {
-    clearTimeout(this.update);
```

Comment:
essa lógica é pra fazer um debounce pra esperar um pouco antes de chamar a rota de frases (aliás, costumava ser 2000ms, não era pra ser 5); sem isso esse componente vai chamar todas as rotas toda vez que uma frase for traduzida (umas 15 vezes por página)